modules/core/src/main/java/org/locationtech/jts/geom/LinearRing.java

```diff
@@ -132,7 +132,7 @@ protected LinearRing copyInternal() {
     return new LinearRing(points.copy(), factory);
   }
 
-  protected LinearRing reverse()
+  public LinearRing reverse()
```

Comment:
Would be better if this was not part of this PR

Reply:
yes it would, let me fix and then rebase this PR.